drivers/SmartThings/philips-hue/src/handlers/attribute_emitters.lua

```diff
@@ -175,6 +175,13 @@ function AttributeEmitters.connectivity_update(child_device, zigbee_status)
     child_device.log.info_with({hub_logs=true}, "Device zigbee status event, marking device online")
     child_device:online()
     child_device:set_field(Fields.IS_ONLINE, true)
+    if driver then
```

Comment:
I only recently realized myself that `.driver` is a field on a device record, so you can actually call `child_device.driver` here instead of passing an argument, unless you want to use that argument as a sentinel value; in which case I think it should be a boolean parameter.